### DISCUSSION TITLE
v1.2.4

### DISCUSSION THREAD

**Discussion by constanline** [constanline/XQuickEnergy#556] [Announcements/公告]
<hr /><em>This discussion was created from the release <a href='https://github.com/constanline/XQuickEnergy/releases/tag/v1.2.4'>v1.2.4</a>.</em>

**Comment by amm10090** [comment]
更新了啥

**Reply by haffers** [reply]
作者说了只做基础维护了噢，具体看这里https://github.com/constanline/XQuickEnergy/discussions/548

**Comment by qingfeng256** [comment]
可以小鸡抽奖吗？

**Comment by yichuimeng** [comment]
下载试试，

**Comment by yichuimeng** [comment]
![Screenshot_2024-06-21-12-30-09-955_com.miui.home.jpg](https://github.com/constanline/XQuickEnergy/assets/168794149/96de1383-b055-47f7-bd7f-031e11de7b76)

激活后，支付宝直接打不开了白屏😂

**Comment by yichuimeng** [comment]
![Screenshot_2024-06-21-12-30-09-955_com.miui.home.jpg](https://github.com/constanline/XQuickEnergy/assets/168794149/96de1383-b055-47f7-bd7f-031e11de7b76)

激活后，支付宝直接打不开了白屏😂

**Reply by DawnAleax** [reply]
+1

**Reply by rkdawn** [reply]
之前作者有说过，去看看之前的文档，需要创建一个/storage/emulated/0/Download/useMedia/文件夹

**Reply by haffers** [reply]
亲测支持最新版支付宝，你更新支付宝试试.当前最新支付宝版本10.6.8.8000

**Reply by GreatMingEmpire** [reply]
請問激活後有啥區別嗎？
怎麼激活呀。

**Reply by doooq** [reply]
> 之前作者有说过，去看看之前的文档，需要创建一个/storage/emulated/0/Download/useMedia/文件夹

兄弟创建个啥文件夹，取啥名字，找不到了，求告知

**Reply by csw121011** [reply]
就在Download文件夹里新建一个useMedia的文件夹

**Comment by li0625** [comment]
支持支付宝最高哪个版本

**Reply by haffers** [reply]
亲测10.6.8.8000

**Comment by komican** [comment]
过期能量可否设置不自动收取？之前默认是不会收取的，现在只要执行就自动收取了

**Comment by csw121011** [comment]
小米的澎湃系统用不了呢，有没有大佬指点一下

**Reply by komican** [reply]
可以用的，我也在用！是magisk模板吧？这个XQuickEnergy V1.2.4就不建议用了，功能比较基础，用其它版本的。比如https://github.com/TKaxv-7S/Sesame-TK，当然还有另一个，但已经闭源了。

**Reply by csw121011** [reply]
我看这个已经没有更新了吗

**Reply by csw121011** [reply]
可以分享一个现在能用吗，谢谢

**Reply by komican** [reply]
> 可以分享一个现在能用吗，谢谢

https://github.com/LazyImmortal/Sesame作者已经重新开源了

**Comment by Jiuridezu** [comment]
不能自动浇水了
